modules/ROOT/pages/configure-notifications.adoc

```diff
@@ -13,22 +13,25 @@ Each email notification includes:
 
 Enable and configure the notifications to automatically send email when the latest version of an API becomes non-conformant. 
 
-NOTE: Notifications are sent only if there is a valid contact email address configured for the selected recipient.
+API Governance sends notifications about conformance only when:
 
-Automated notifications are sent for each ruleset that has a new conformance failure when the conformance status changes during validation. A notification is sent for a specific API only the first time that version of the API fails conformance to a specific ruleset. 
+* One or more active profiles targets the API.
+* A valid contact email address is configured for the selected recipient.
+* An API's conformance status changes to fail because of a change in the API or applied rulesets.
+* The first time a specific version of an API fails conformance to a specific ruleset. 
 
 Following are the changes that might result in notifications: 
 
-. Updating a governance profile, including adding or removing rulesets 
-. Publishing a new version of an API 
-. Changing tags on an API 
-. Updating governance rulesets 
+* Updating an active governance profile, including adding or removing rulesets 
+* Publishing a new version of an API 
+* Changing tags on an API 
+* Updating governance rulesets 
```

Comment:
activating a  draft profile will also result in notifications

Reply:
if configured, etc etc. etc

Reply:
Added